languages/csharp/exercises/concept/nullability/.docs/instructions.md

```diff
@@ -0,0 +1,30 @@
+In this exercise you'll be writing code to helping printing name
+badges for factory employees.
+
+Employees have an id, a name and a department name. Badge labels are
+formatted as follows: `"[id] - [name] - [DEPARTMENT]"`
+
+### 1. Generate the badge label from employee properties
+
+Writing a static method called `Badge.Label()` to compute the string
+to be printed on the badge:
```

Comment:
Slightly more concise and in line with how [existing exercises phrase things](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/instructions.md#1-get-message-from-a-log-line):

```suggestion
Implement the `Badge.Label()` method to return an employee's badge label:
```

Also, people won't be writing the method but implementing it, as there already is a stub.